prow/scripts/cluster-integration/helpers/generate-and-export-TLS-cert.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/library.sh"
+
+if [ -z "${DOMAIN}" ] ; then
+    echo "ERROR: DOMAIN is not set"
+    exit 1
+fi
+
+shout "Generate lets encrypt certificate"
```

Comment:
here we generate let's encrypt certs, so the name of this file should indicate what kind of cert is generated

Reply:
Renamed the file as `generate-and-export-letsencrypt-TLS-cert.sh`